dev/fattest.simplicity/bnd.bnd

```diff
@@ -59,6 +59,7 @@ test.project: true
 	commons-httpclient:commons-httpclient;version=3.1,\
 	com.ibm.websphere.javaee.jsonp.1.0;version=latest,\
 	com.ibm.websphere.javaee.servlet.3.0;version=latest,\
+	com.ibm.websphere.jakartaee.servlet.5.0;version=latest,\
```

Comment:
Lets remove the compile-time dependencies on servlet 3 and servlet 5, and instead just use reflection to load/test the classes in TestServletProcessor.java

Reply:
I did not make this change. Upon further investigation I don't think this change is feasible due to a reflection restriction involving classloading.  Within TestServletProcessor we can remove the build dependencies on classes \<platform\>.servlet.http.HttpServlet, but not on classes \<platform\>.servlet.annotation.WebServlet.  The getQueryPaths() method computes the servlet query path from the value and urlPatterns methods (attributes) of the WebServlet annotation.  It's possible to invoke these methods using reflection, but in the FAT runtime the following pattern fails with an exception indicating the _object is not an instance of declaring class_, which means the method (m) and the target class (webServletClass) are not loaded by the same classloader. I don't see how to alleviate that restriction.

`Class<? extends Annotation> webServletClass = 
     Class.forName("platform.servlet.annotation.WebServlet");

 Annotation webServlet = anno.servlet().getAnnotation(webServletClass);

 m = webServletClass.getDeclaredMethod("value");

 webServletValue = (String[]) m.invoke(webServletClass, (Object[]) null);`